src/os.cpp

```diff
@@ -1719,6 +1765,15 @@ static uint8_t utf8CodepointSequenceLength(uint32_t c) {
     zig_unreachable();
 }
 
+// Ported from std.unicode.utf8ByteSequenceLength
+static uint8_t utf8ByteSequenceLength(uint8_t first_byte) {
+    if (first_byte < 0b10000000) return 1;
+    if ((first_byte & 0b11100000) == 0b11000000) return 2;
+    if ((first_byte & 0b11110000) == 0b11100000) return 3;
+    if ((first_byte & 0b11111000) == 0b11110000) return 4;
```

Comment:
Switching over `clzll(~first_byte)` instead here like is done in the stdlib yielded incorrect results on win64.  
Specifically for `first_byte = 0xC3 =0b11000011` I was getting `0` back instead of `2`.